tests/test_galaxy_autoencoder.py

```diff
@@ -2,7 +2,7 @@ def test_galaxy_autoencoder_toy_gaussian(model_setup, devices):
     use_cuda = devices.use_cuda
     overrides = {
         "model": "galaxy_net",
-        "model.residual_delay_n_steps": 50 if use_cuda else 0,
+        "+model.residual_delay_n_steps": 50 if use_cuda else 0,
```

Comment:
what is the purpose of this flag? does it delay using the residual part of the VAE when training until after 50 steps?

Reply:
also, why did you remove it from the config? (so now you have use the `+`)

Reply:
> I removed the flag because I think we should only specify arguments in the Hydra configs when we explicitly want to override the defaults from the "target" class.

Good point, otherwise the default in the config and the default of the class might get out of sync